### ISSUE TITLE
Reverse dependency checks?

### ISSUE THREAD

**Issue by wlandau** [r-universe-org/help#362]
Would it be feasible to run reverse dependency checks when a package is updated? Maybe this is too computationally demanding for the development versions of packages, but maybe it would help with releases? C.f. #135, #149, #289.

**Comment by jeroen** [comment]
Thanks yes that is in the pipeline. See also: https://github.com/r-universe-org/help/issues/364

**Comment by wlandau** [comment]
Awesome!

Would it be possible to opt into revdep checks for an entire universe? In the case of https://github.com/r-releases, it would be cumbersome to manually activate revdep checks for each new added package.

**Comment by wlandau** [comment]
As I mention in https://github.com/r-universe-org/help/issues/369#issue-2164948647, I don't actually think revdep checks are necessary for `r-releases`. But it would really help to rerun the checks right when a strong dependency updates, like you originally suggested in https://github.com/r-universe-org/help/issues/368#issuecomment-1972767243. I posted a separate issue for this in #369. Closing this one in favor of #369 and #362.
